Automattic/wp-calypso
# Update redirection to jetpack complete after authorization

Related to #
1203759331596262-as-1203791007934484/f

## Proposed Changes
This PR changes redirects after Jetpack plugin activation (Authorization) to navigate to jetpack complete page. This should only appear for non-multisite and only when jetpack plugin activated itself, not individual products.
This PR also adds tests to check the behaviour.

## Testing Instructions
- Download PR locally and run it `yarn start` or use Live link provided below
- Create a Single site in JN with a pre-installed jetpack plugin
- Press `setup jetpack`  and you'll be navigated to `authorize` page
- change `https://wordpress.com/`  in URL to either `http://calyspo.localhost:3000` or 'Live link' and add `flags=jetpack/offer-complete-after-activation` param in the URL
-  hit `Approve` button and you should be navigated to Complete page: `http://calypso.localhost:3000/jetpack/connect/plans/complete`

- Create a Multi site in JN with a pre-installed jetpack plugin
- Press `setup jetpack`  and you'll be navigated to `authorize` page
- change `https://wordpress.com/`  in URL to either `http://calyspo.localhost:3000` or 'Live link' and add `flags=jetpack/offer-complete-after-activation` param in the URL
-  hit `Approve` button and you should be navigated to pricing page: `http://calypso.localhost:3000/jetpack/connect/plans/`

## Pre-merge Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203791007934484
